gapis/service/service.go

```diff
@@ -134,6 +135,9 @@ type Service interface {
 	// This is a debug API, and may be removed in the future.
 	Profile(ctx context.Context, pprof, trace io.Writer, memorySnapshotInterval uint32) (stop func() error, err error)
 
+	// Status starts resolving status events. It calls f for every update and m for every status update.
```

Comment:
Comment: m for every memory status update?

Reply:
Fixed